### PR TITLE
Website updates for the 3.0 beta1 release

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

**Review by levitte — APPROVED** [review]

File: news/newsflash.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@
 # headings.  URL paths must all be absolute.
 Date: Item
 
+17-Jun-2021: New Blog post: <a href="/blog/blog/2021/06/17/OpenSSL3.0ReleaseCandidate/">OpenSSL 3.0 Release Candidate</a>
```
Comment (levitte) [inline news/newsflash.txt:8]: I assume this PR won't be merged before that blog post is in place

Comment (mattcaswell) [reply news/newsflash.txt:8]: Correct

**Review by paulidale — APPROVED** [review]

File: news/newsflash.txt (hunk 1 above)
Comment (slel) [reply news/newsflash.txt:8]: That blog post is in place now.

Comment (mattcaswell) [reply news/newsflash.txt:8]: This PR was in fact merged straight after the beta release, but I forgot to close this PR. Closing it now.

**Pull request closed without merging**